src/meson.build

```diff
@@ -18,7 +18,8 @@ if git_exe.found()
     # Get full description based on git tag API version ('v0.2-22-g3d8867c-dirty')
     git_f_api = run_command(
         git_exe, '-C', meson.current_source_dir(),
-            'describe', '--tags', '--always', '--dirty', '--match', 'v*'
+            'describe', '--tags', '--always', '--dirty', '--match', 'v*',
+        check: true
```

Comment:
This "check: true" will probably break the build if you have a `git` executable, but your source code is embedded in some other weird git repo (like vcpkg does it). Note that two lines later we explicitly handle this situation.

Reply:
Good point, I actually did not pay attention to that. Changed!

Reply:
See also

d41fd54 `meson: Allow configuration without working git`

:grimacing: